extensions/PhabBugz/lib/Util.pm

```diff
@@ -428,7 +430,9 @@ PROJECT: foreach my $project (@review_projects) {
 
     # If we still have not found a reviewer and there is no member that was the
     # last reviewer, then just pick the first member in the list
-    if (!$last_reviewer_phid) {
+    if (!$last_reviewer_phid
```

Comment:
As an aside: do we actually need this check? If we removed this, the first reviewer would still be set in the loop over the group members below.